src/compiler/checker.ts

```diff
@@ -14637,7 +14639,9 @@ namespace ts {
                     location = location.parent;
                 }
 
-                copySymbols(globals, meaning);
+                if (includeAllGlobalSymbols) {
+                    copySymbols(globals, meaning);

```

Comment:
I don't understand why we would do this; presumably this means that a React component defined at the global scope wouldn't be put in the completion list.